common/djangoapps/oauth_exchange/views.py

```diff
@@ -35,3 +43,26 @@ def post(self, request, _backend):
             edx_access_token = self.create_access_token(request, user, scope, client)
 
         return self.access_token_response(edx_access_token)
+
+
+class SessionCookieExchangeView(APIView):
+    """
+    View for exchanging an access token for session cookies
+    """
+    authentication_classes = (OAuth2AuthenticationAllowInactiveUser,)
+    permission_classes = (permissions.IsAuthenticated,)
+
+    @method_decorator(csrf_exempt)
+    def get(self, request):
+        request.user.backend = get_backend_for_user(request.user)
```

Comment:
The value of the backend attribute should be the path of the backend class (https://github.com/django/django/blob/1.4.20/django/contrib/auth/__init__.py#L52).